docs/scoring.md

```diff
@@ -0,0 +1,64 @@
+## Scoring volumes
+
+We suggest to adhere to the list of scoring quantities and scoring volumes described below. This will allow to compare the results obtained using different MC codes.
+
+### Target volume
+
+This volume represent the detector positioned in the PMMA holder. Center of the volume is equivalent to the isocenter position. The volume is large enough to cover the whole detector.
+
+`[-2.5, 2.5] x [-2.5, 2.5] x [-0.1, 0.1] mm³`
+
+Lateral extension of all the fields in treament plans is twice the size of the target volume. Plans cover the PTV of 10 x 10 x 10 cm³.
+
+This is the target volume for plan2:
+![Target for plan2](plan2-target.png)
+
+### Narrow depth profile
+
+To score depth profile of basic quantities we suggest to use a narrow volume of 2 x 2 x 20.5 cm³. The volume is centered at the isocenter position. 
+
+`[-1.0, 1.0] x [-1.0, 1.0] x [-10.25, 10.25] cm³`
+
+The volume should be divided into 205 bins of 1 mm thickness along the Z axis.
+
+The span of the volume along Z axis: 20.5 cm covers the 20 cm of the solid water phantom and 0.5 cm of the detector plate.
+
+This is the narrow profile for plan2:
+![Alt text](plan2-narrow.png)
+
+### Lateral map
+
+Two dimensional map is useful to check the spot positions at the isocenter plane. We suggest to use a thin slab covering whole solid water phantom in lateral direction and 5 mm in depth.
+
+`[-15.0, 15.0] x [-15.0, 15.0] x [-0.25, 0.25] cm³`
+
+This is the lateral map for plan2:
+![Alt text](plan2-lateral.png)
+
+### Longitudal map
+
+Two dimensional map useful to check the beam divergence. We suggest to use a thin 2cm slab positioned in XZ plane:
+
+`[-15.0, 15.0] x [-1.0, 1.0] x [-10.25, 10.25] cm³`
+
+This is the longitudal map for plan2:
+![Alt text](plan2-longitudal.png)
+## Scoring quantities
+
+### Target volume
+
+- dose deposited by all particles
+- fluence of primary protons
+- all types of averaged linear energy transfer (LET)
```

Comment:
since this is the scoring section, maybe be a bit more explicit?

for LET lets refer to a subsection:
1) dose-, track average
2) primaries, all protons, all particles
3) in water at density 1.000, LET in medium at medium density

and link to our paper Kalholm et al for reference.